.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
+*.py text eol=lf
+
+# Declare files that will always have CRLF line endings on checkout.
+*.sln text eol=crlf
```

Comment:
Please, clear this doubt for me: aren't .sln files tool-specific? That is, isn't this line for a feature specific of VS code? Nothing against VS code, I'm asking because I'd like to keep Nodezator (and any app/game of the Indie Python project) tool-agnostic, that is, I'd like it to not have any traces of any specific tool on it.

That applies even to me, the maintainer. There are tool-specific configurations and files that I keep out of Nodezator's configs/source control because no one should be forced to use the tools I use nor having to look at settings for tools that they don't even use.

Reply:
Instead, tool-specific configs/settings should, whenever possible, be set in the developer's own environment, no in the repository.

For instance, not related to .gitattributes but .gitignore, my system-wide .config file (the one in my home folder) points to a system-wide .gitignore file (also in my home folder) that ignores specific files related to my workflow in any git project in my pc.

If we were to add such settings to the repository we'd end up with a lot of lines added for all the different editors/IDEs people use for their own personal workflows.